builtin/logical/transit/path_encrypt.go

```diff
@@ -63,46 +65,32 @@ func (b *backend) pathEncryptWrite(
 	// Decode the context if any
 	contextRaw := d.Get("context").(string)
 	var context []byte
+	var err error
 	if len(contextRaw) != 0 {
-		var err error
```

Comment:
re: returning w/ stack variables, in this case I'd just name `err` as the variable for the return parameter and would use that throughout this method.